arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -1399,6 +1399,9 @@ Result RocksDBCollection::insertDocument(transaction::Methods* trx,
     return res.reset(TRI_ERROR_DEBUG);
   }
 
+  ConcurrencyControlToggler cct(mthds, state->isOnlyExclusiveTransaction());
+  cct.setConcurrencyControl(false);
```

Comment:
I think will be nice to move it the ctor, because re-enabling in the dtor

Reply:
Done